src/parakeet/Pro/Driver.cpp

```diff
@@ -70,12 +59,21 @@ namespace Pro
     {
         return SW_SET_BIAS_PREFIX + std::to_string(bias) + SW_SET_BIAS_POSTFIX;
     }
+    
+    Driver::Driver()
+    {
+        this->registerUpdateThreadCallback(std::bind(&Driver::serialUpdateThreadFunction, this));
+
+        serialPortDataBuffer = new unsigned char[SERIAL_MESSAGE_DATA_BUFFER_SIZE];
```

Comment:
Seems like there's no reason to use a pointer here. It looks like we have the same size buffer for the lifetime of this owning object.